components/frontend/src/source/SourceEntityDetails.js

```diff
@@ -27,12 +27,11 @@ entityStatusOption.propTypes = {
 }
 
 function entityStatusOptions(entityType, report) {
-    const desiredResponseTimes = report?.desired_response_times ?? {}
     const unconfirmedSubheader = `This ${entityType} should be reviewed to decide what to do with it.`
-    const confirmedSubheader = `This ${entityType} has been reviewed and should be dealt with within ${desiredResponseTimes["confirmed"]} days.`
-    const fixedSubheader = `Ignore this ${entityType} for ${desiredResponseTimes["fixed"]} days because it has been fixed or will be fixed shortly.`
-    const falsePositiveSubheader = `Ignore this ${entityType} for ${desiredResponseTimes["false_positive"]} days because it's been incorrectly identified as ${entityType}.`
-    const wontFixSubheader = `Ignore this ${entityType} for ${desiredResponseTimes["wont_fix"]} days because it will not be fixed.`
+    const confirmedSubheader = `This ${entityType} has been reviewed and should be dealt with within ${getDesiredResponseTime(report, "confirmed")} days.`
+    const fixedSubheader = `Ignore this ${entityType} for ${getDesiredResponseTime(report, "fixed")} days because it has been fixed or will be fixed shortly.`
+    const falsePositiveSubheader = `Ignore this ${entityType} for ${getDesiredResponseTime(report, "false_positive")} days because it has been incorrectly identified as ${entityType}.`
```

Comment:
Consider rewording to `incorrectly identified as a ${entityType}` or `incorrectly identified as "${entityType}"`

Reply:
Hmm "as a" won't work for issues, for example, because it would need to be "as an".  

I think the quotes would be an improvement, but instead around the first occurrence of the entity type, since that's where you'd use air quotes if you had to say the sentence out loud:

> Ignore this "violation" for 180 days because it has been incorrectly been identified as violation.